Makefile

```diff
@@ -19,6 +19,10 @@ BUILDTAGS += $(EXTRA_BUILDTAGS)
 
 COMMIT ?= $(shell git describe --dirty --long --always)
 VERSION := $(shell cat ./VERSION)
+VERSION_EXTRA :=
+ifneq (,$(VERSION_EXTRA))
```

Comment:
No need to have the extra complexity of ifneq / endif; adding an empty string have no effect.

Reply:
Right, now that the separator should be in the variable, no need for this. Thanks!